Makefile

```diff
@@ -412,6 +418,10 @@ test-k8s-nuctl:
 	NAMESPACE=$(NAMESPACE) \
 	./test/k8s/ci_assets/nuctl.sh
 
+.PHONY: ensure-builder
+ensure-builder:
```

Comment:
```suggestion
build-builder:
```
This is rebuilding, which is important semantically (needs to completely rebuild in case of any go.mod changes)